.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
+
+install_status=1
+
+SRC="${PWD}/_out/build_install"
+
+# Expect the host filesystem to be mounted in following path.
+HOST_MOUNT=/host
+
+# Some files are copied over /usr which on Red Hat CoreOS (rhcos) is mounted
+# read-only by default. So re-mount it as read-write, otherwise files won't
+# get copied.
+mount -o remount,rw $HOST_MOUNT/usr
+
+# The host's '/opt' and '/usr/local' are symlinks to, respectively, '/var/opt'
+# and '/var/usrlocal'. Adjust the installation files accordingly.
+#
+cd $SRC
+if [ -d 'opt' ]; then
+	mkdir -p var
+	mv opt var
+fi
+
+if [ -d "usr/local" ]; then
+	mkdir -p var
+	mv usr/local var/usrlocal
+fi
+
+# Copy the installation over the host filesystem.
+rsync -a * $HOST_MOUNT
```

Comment:
Should that be `.` rather than `*` in case of hidden files?

Reply:
Probably better, although installing hidden files on the host is a bad idea.

Reply:
yeah, I am not expecting hidden files but I changed as suggested in the v4 of this PR.